src/Backupper/BackupperFactory.php

```diff
@@ -22,6 +24,7 @@ public function __construct(
         private ManagerRegistry $doctrineRegistry,
         private array $backupperBinaries,
         private array $backupperOptions = [],
+        private ?LoggerInterface $logger = null,
```

Comment:
You should make it non-nullable and create a null instance here, no ? That's not mandatory but I dislike dangling nullables for such services. I don't know if this would make any sense to instrument the factories themselves ?

Reply:
I chose to make it "nullable" to avoid adding a useless `NullLogger` into the backupper's `ChainLogger`. I usually don't like having to deal with `null` values as well, but in this case of a short factory I consider that this is acceptable.

Reply:
OK ! That makes sense.